### ISSUE TITLE
Ghoul's paralysis does not expire

### ISSUE THREAD

**Issue by grannypron** [grannypron/uaf_levels#73]
After a ghoul attacked two different PCs of mine, the message "so and so is paralyzed" appeared, but they were able to move and attack just fine for that round and rounds after it.

**Comment by grannypron** [comment]
The relevant SA looks like:

```
name = IsParalyzed
[CharDisplayStatus] = $SET_HOOK_PARAM( 7,"Paralyzed");
. . . 
[IsCombatReady] = $RETURN "N";
. . . 

```
IsCombatReady is supposed to be called and should tell the engine that the player can't move.

**Comment by grannypron** [comment]
OK so I think two things are going on here:

1) The paralyze effect is not making the player stop from moving
2) when a player gets hit, sometimes they get the message that "so-and-so is paralzyed", even when the paralyze effect is not displayed.  I think this might be when the player makes the saving throw.

I was able to fix 1 above by seeing that this wasn't happening in the TutorialDesign and updating the spell_Paralysis SA from TutorialDesign, but 2 is still happening.  2 may be a known bug, idk.

**Comment by grannypron** [comment]
Also, I think the effect is probably not wearing off after the expiration time.   The expiration time will have to be calculated in spell_Paralysis and then parameter 2 will have to be set to that expiration time (in minutes).  Then we should add back in the [StartCombatRound] hook (not sure why I removed it, except that maybe there was an error because the SA_PARAM() returned a string.

**Comment by grannypron** [comment]
Checked and paralysis is not wearing off.  When setting the expiration date, Stephen Lee's FAQ says it lasts for 1d6+2 rounds.

**Comment by grannypron** [comment]
Somehow, we will have to differentiate between this spell_Paralysis and other paralyzation attacks, like from Driders or Mummy's fear auras.

**Comment by grannypron** [comment]
Since #142 is filed, I am now making this ticket about the ghoul paralysis affect wearing off after 1d6+2

**Comment by grannypron** [comment]
Implemented paralysis expiration with commit c7701a9
